### PR TITLE
feat: add support for VRS 1.3 RC

### LINKED ISSUE #58: Add support for VRS 1.3.X
See [this](https://github.com/ga4gh/vrs/pull/418) PR

### PR DESCRIPTION
**Pull request by korikuzma**
Close #58 

Notes:
- I used [datamodel-code-generator](https://github.com/koxudaxi/datamodel-code-generator/) as a reference. It gets all the basic things we need, but misses some validation/constraints in fields
- Updated the README for the VRS/VRSATILE branches associated to vrsatile-pydantic's version / branch

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -15,3 +15,4 @@ flake8-docstrings = "*"
 pre-commit = "*"
 coveralls = "*"
 coverage = "*"
+ipykernel = "*"
```
Comment (jsstevenson) [inline Pipfile:18]: ?

Comment (korikuzma) [reply Pipfile:18]: Running jupyter notebook in vs-code

File: README.md [hunk 2]
```diff
@@ -3,8 +3,11 @@ Translation of the GA4GH [VRS](https://vrs.ga4gh.org/en/stable/) and [VRSATILE](
 
 The ga4gh/vrsatile/pydantic repo depends on VRS and VRSATILE models, and therefore each ga4gh.vrsatile.pydantic package on PyPI uses a particular version of VRS and VRSATILE. The correspondences between the packages may be summarized as:
 
-- **0.0.X ~ 1.2.X**: ga4gh.vrsatile.pydantic 0.0.X tracks [VRS 1.2.X](https://vrs.ga4gh.org/en/1.2.1/) and the [VRSATILE latest](https://vrsatile.readthedocs.io/en/latest/)
-- **0.1.X ~ metaschema-updates**: ga4gh.vrsatile.pydantic 0.1.X tracks VRS and VRSATILE metaschema-updates
+| ga4gh.vrsatile.pydantic branch | ga4gh.vrsatile.pydantic version | VRS version | VRSATILE version |
```
Comment (jsstevenson) [inline README.md:6]: tables. love it.

File: Pipfile (hunk 1 above)
Comment (jsstevenson) [reply Pipfile:18]: By default I'd err towards removing it as a dependency but I don't feel super strongly

**Review by jsstevenson — APPROVED** [review]

File: Pipfile (hunk 1 above)
Comment (korikuzma) [reply Pipfile:18]: Sounds good. We have it in almost all of our other projects too 😬 😅

**Pull request merged**